src/tools/pontoon/search_filters.md

```diff
@@ -55,3 +55,20 @@ For specific projects it’s also possible to filter strings based on *tags*. Ta
 In this case, there are 10 tags defined for the project (highlighted in red). Near each tag there is a representation of the priority: like for projects, it goes from 5 stars (highest priority) to 1 star (lowest priority).
 
 Note: translation time, translation authors, and tags are not available when `All Projects` is selected as a [resource](glossary.md#resource).
+
+## Options
+
+In addition to all of the search filters listed above, Pontoon also provides **search options**, allowing users to refine search results. Search options can be accessed by clicking the magnifying glass icon on the right side of the search box.
+
+![Options](../../assets/images/pontoon/search_filters/options.png)
+
+Search options include:
+* **Match case**: Exactly match the casing of the search term.
+* **Match whole word**: Similar to surrounding with double quotes, exactly match the search term (case insensitive).
+* **Include string identifiers**: Expand the search to include string identifiers (keys).
+* **Include rejected translations**: Expand the search to include translations that have been rejected.
+* **Exclude source strings**: Exclude any matches that appear within source strings.
```

Comment:
```suggestion
* **Exclude source strings**: Avoid searching for matches in source strings.
```

The current wording is potentially incorrect: we don't exclude strings that have matches in the source (but might have matches in translations).